src/pages/en/migrate.md

```diff
@@ -28,6 +28,29 @@ const canonicalURL = Astro.canonicalURL;
 const canonicalURL = new URL(Astro.url.pathname, Astro.site);
 ```
 
+### Changed: Scoped CSS specificity
+
+Specificity will now be preserved in scoped CSS styles. This change will cause most scoped styles to _happen_ to take precedence over global styles. But, this behavior is longer explicitly guaranteed.  
+
+```diff
+// Before: Astro appends .astro-XXXXXX, increasing specificity. 
+<style>
+- div { color: red } // (0,0,1 specificity) 
++ div.astro-XXXXXX { color: red } // (0,1,1 specificity).
+</style>
+
+// After: appends :where(.astro-XXXXXX), maintaining the authored specificity. 
+<style>
+- div { color: red } // (0,0,1 specificity) 
++ div:where(.astro-XXXXXX) // (0,0,1 specificity).
+</style>
+```
```

Comment:
I’ve tried to break up and rewrite this code sample to explain slightly more clearly exactly what is going on here — I found using a diff for input/output not super obvious in this case.

I’ve also updated the code samples to use the correct format for CSS comments and added a link to docs for the `:where` pseudo-class as its on the newer side of CSS syntax.

````suggestion
Technically, this is accomplished using [the `:where()` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) instead of using classes directly in Astro’s CSS output.

Let’s use the following style block in an Astro component as an example:

```astro
<style>
div { color: red; } /* 0-0-1 specificity */
</style>
```

Previously, Astro would transform this into the following CSS, which has a specificity of `0-1-1` — a higher specificity than the source CSS:

```css
div.astro-XXXXXX { color: red; } /* 0-1-1 specificity */
```

Now, Astro wraps the class selector with `:where()`, maintaining the authored specificity:

```css
div:where(.astro-XXXXXX) { color: red; } /* 0-0-1 specificity */
```
````